request.js

```diff
@@ -240,6 +239,14 @@ function Request (options) {
 
 util.inherits(Request, stream.Stream)
 
+// Debugging
+Request.debug = process.env.NODE_DEBUG && /\brequest\b/.test(process.env.NODE_DEBUG)
+function debug() {
```

Comment:
nit: I'd expect to see the debug function up with the other inlined functions, instead of here in the Request prototype methods